### PR TITLE
Feature/eu 6403 add user confirm

### PR DESCRIPTION
**Pull request by aamirhm**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-6403

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/app/components/info-message/info-message.component.spec.ts [hunk 1]
```diff
@@ -1,20 +1,21 @@
-import { Component, Input, ViewChild } from '@angular/core';
+import { Component, CUSTOM_ELEMENTS_SCHEMA, Input, ViewChild } from '@angular/core';
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
-
-import {InfoMessage, InfoMessageType} from '../../enums';
-import { WorkAllocationComponentsModule } from '../work-allocation.components.module';
+import { InfoMessage } from 'src/app/shared/enums/info-message';
```
Comment (DavidJayakumar) [inline src/app/components/info-message/info-message.component.spec.ts:3]: Path should be relative '../'. Should not start with 'src'. Also noticed in few other places below

File: src/app/components/info-message/info-message.component.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import {Component, Input} from '@angular/core';
-
-import {InfoMessage, InfoMessageType} from '../../enums';
+import { InfoMessage } from 'src/app/shared/enums/info-message';
```
Comment (DavidJayakumar) [inline src/app/components/info-message/info-message.component.ts:2]: src path

File: src/app/containers/info-message-container/info-message-container.component.ts [hunk 3]
```diff
@@ -1,8 +1,7 @@
 import { Component, OnInit } from '@angular/core';
 import { NavigationEnd, NavigationStart, Router } from '@angular/router';
-import { InfoMessageCommService } from '../../../app/shared/services/info-message-comms.service';
-
-import { InformationMessage } from '../../models/comms';
+import { InformationMessage } from 'src/app/shared/models';
```
Comment (DavidJayakumar) [inline src/app/containers/info-message-container/info-message-container.component.ts:3]: src path

File: src/cases/components/case-task/case-task.component.ts [hunk 4]
```diff
@@ -2,11 +2,12 @@ import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
 import { Router } from '@angular/router';
 
 import { AlertService } from '@hmcts/ccd-case-ui-toolkit';
+import { InfoMessage } from 'src/app/shared/enums/info-message';
```
Comment (DavidJayakumar) [inline src/cases/components/case-task/case-task.component.ts:5]: src path

**Review by Surekha-HMCTS — APPROVED** [review]

File: src/work-allocation/containers/available-tasks/available-tasks.component.ts [hunk 5]
```diff
@@ -1,9 +1,11 @@
 import { Component } from '@angular/core';
+import { InfoMessage } from 'src/app/shared/enums/info-message';
```
Comment (DavidJayakumar) [inline src/work-allocation/containers/available-tasks/available-tasks.component.ts:2]: src path

File: src/work-allocation/containers/task-action/task-action-container.component.spec.ts [hunk 6]
```diff
@@ -8,13 +8,14 @@ import { RouterTestingModule } from '@angular/router/testing';
 import { PaginationModule } from '@hmcts/ccd-case-ui-toolkit';
 import { SessionStorageService } from '@hmcts/ccd-case-ui-toolkit/dist/shared/services';
 import { Observable, throwError } from 'rxjs';
+import { InfoMessage } from 'src/app/shared/enums/info-message';
```
Comment (DavidJayakumar) [inline src/work-allocation/containers/task-action/task-action-container.component.spec.ts:11]: src path

File: src/work-allocation/containers/task-assignment-confirm/task-assignment-confirm.component.spec.ts [hunk 7]
```diff
@@ -7,13 +7,15 @@ import { RouterTestingModule } from '@angular/router/testing';
 import { SessionStorageService } from '@hmcts/ccd-case-ui-toolkit/dist/shared/services';
 import { ExuiCommonLibModule } from '@hmcts/rpx-xui-common-lib';
 import { Observable, of, throwError } from 'rxjs';
+import { InfoMessage } from 'src/app/shared/enums/info-message';
```
Comment (DavidJayakumar) [inline src/work-allocation/containers/task-assignment-confirm/task-assignment-confirm.component.spec.ts:10]: src path

File: src/work-allocation/containers/task-list-wrapper/task-list-wrapper.component.ts [hunk 8]
```diff
@@ -28,6 +28,8 @@ import {
 } from '../../services';
 import { getAssigneeName, handleFatalErrors, WILDCARD_SERVICE_DOWN } from '../../utils';
 import * as fromActions from '../../../app/store';
+import { InfoMessageType } from 'src/app/shared/enums/info-message-type';
```
Comment (DavidJayakumar) [inline src/work-allocation/containers/task-list-wrapper/task-list-wrapper.component.ts:31]: src path

File: src/work-allocation/containers/work-case-list-wrapper/work-case-list-wrapper.component.ts [hunk 9]
```diff
@@ -6,14 +6,16 @@ import { FeatureToggleService, FilterService, FilterSetting } from '@hmcts/rpx-x
 import { select, Store } from '@ngrx/store';
 import { Observable, of, Subscription } from 'rxjs';
 import { debounceTime, filter, mergeMap, switchMap } from 'rxjs/operators';
+import { InfoMessage } from 'src/app/shared/enums/info-message';
```
Comment (DavidJayakumar) [inline src/work-allocation/containers/work-case-list-wrapper/work-case-list-wrapper.component.ts:9]: src path

File: src/work-allocation/models/common/route-data.ts [hunk 10]
```diff
@@ -1,4 +1,5 @@
-import { InfoMessage, TaskActionType } from '../../enums';
+import { InfoMessage } from 'src/app/shared/enums/info-message';
```
Comment (DavidJayakumar) [inline src/work-allocation/models/common/route-data.ts:1]: src path

**Review by DavidJayakumar — COMMENTED** [review]
Change src path references to releative path

File: src/app/containers/info-message-container/info-message-container.component.spec.ts [hunk 11]
```diff
@@ -1,11 +1,11 @@
 import { Component, Input, ViewChild } from '@angular/core';
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
-import { InfoMessageCommService } from '../../../app/shared/services/info-message-comms.service';
 
-import { WorkAllocationComponentsModule } from '../../components/work-allocation.components.module';
-import { InfoMessageContainerComponent } from '..';
 import { Router } from '@angular/router';
 import { RouterTestingModule } from '@angular/router/testing';
+import { InfoMessageCommService } from 'src/app/shared/services/info-message-comms.service';
```
[the remark this answers is not in the document]
Comment (aamirhm) [reply src/app/containers/info-message-container/info-message-container.component.spec.ts:6]: Updated.

File: src/app/containers/info-message-container/info-message-container.component.ts (hunk 3 above)
Comment (aamirhm) [reply src/app/containers/info-message-container/info-message-container.component.ts:3]: Updated.

File: src/work-allocation/containers/task-action/task-action-container.component.ts [hunk 12]
```diff
@@ -1,17 +1,19 @@
 import { Component, OnInit } from '@angular/core';
 import { ActivatedRoute, Router } from '@angular/router';
 import { SessionStorageService } from '@hmcts/ccd-case-ui-toolkit/dist/shared/services';
+import { InfoMessage } from 'src/app/shared/enums/info-message';
```
[the remark this answers is not in the document]
Comment (aamirhm) [reply src/work-allocation/containers/task-action/task-action-container.component.ts]: Updated.

File: src/work-allocation/containers/task-assignment-confirm/task-assignment-confirm.component.spec.ts (hunk 7 above)
Comment (aamirhm) [reply src/work-allocation/containers/task-assignment-confirm/task-assignment-confirm.component.spec.ts]: Updated.

File: src/work-allocation/containers/work-case-list-wrapper/work-case-list-wrapper.component.ts (hunk 9 above)
Comment (aamirhm) [reply src/work-allocation/containers/work-case-list-wrapper/work-case-list-wrapper.component.ts]: Updated.

File: src/work-allocation/models/common/route-data.ts (hunk 10 above)
Comment (aamirhm) [reply src/work-allocation/models/common/route-data.ts]: Updated.

**Pull request merged**